code/modules/vending/security.dm

```diff
@@ -82,6 +82,10 @@
 	name = "security assistant voucher"
 	icon_state = "security_voucher_assistant"
 
+/obj/item/security_voucher/brig_physician
+	name = "security brig physician voucher"
+	icon_state = "security_voucher_brig_phyisician"
```

Comment:
The icon state name has a typo - currently `physician` is typo'd as `phyisician`.